lib/shared/constants.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:flutter/material.dart';
+
```

Comment:
Maybe the file should be renamed styling, or something like that? It would better describe the content.

Reply:
This still needs to be addressed.